doc/api/net.md

```diff
@@ -226,6 +226,11 @@ added: v0.1.90
 Begin accepting connections on the specified `port` and `hostname`. If the
 `hostname` is omitted, the server will accept connections on any IPv6 address
 (`::`) when IPv6 is available, or any IPv4 address (`0.0.0.0`) otherwise.
+
+*Note*: Depending on the operating system, listening to an IPv6 address such
+as `'::'` may cause the `net.Server` to listen on an equivalent IPv4 address
```

Comment:
So if it's just `::` and `::1` that exhibit this behaviour, then we should probably just be explicit, something like:


> ... listening on one of the IPv6 addresses `::` or `::1` may cause the `net.Server` to listen on the equivalent IPv4 address (`0.0.0.0` or `127.0.0.1`) ...